cloudformation/alarms.yaml

```diff
@@ -27,7 +27,11 @@ Resources:
       Statistic: Sum
       Threshold: 1
       TreatMissingData: notBreaching
-      Unit: Count  
+      Unit: Count
+      OKActions:
+        - !Ref CloudWatchAlarmSNSTopic
```

Comment:
Could you do a find/replace on `CloudWatchAlarmSNSTopic` -> `FhirWorksAlarmSNSTopic`